src/Directory.Build.props

```diff
@@ -107,8 +107,13 @@
 	</PropertyGroup>
 
 	<PropertyGroup Condition="$(_IsUWP)">
+		<DefineConstants>$(DefineConstants);__WINDOWS__</DefineConstants>
```

Comment:
Isn't there already a "WINDOWS" constant defined that we can use?

Reply:
WINDOWS seems to be only for UWP